workspaces/todo/package.json

```diff
@@ -30,9 +30,9 @@
     "directory": "workspaces/todo"
   },
   "devDependencies": {
-    "@backstage/cli": "^0.26.2",
+    "@backstage/cli": "^0.26.3",
     "@backstage/e2e-test-utils": "^0.1.1",
-    "@backstage/repo-tools": "^0.7.2",
+    "@backstage/repo-tools": "^0.6.2",
```

Comment:
what's up here

Reply:
This is at the workspace level, I think I added the version manually before, and then added one to the script for creating at the workspace, can update the versions so they're the latest.

Reply:
aight